packages/amplify-cli/src/init-steps/s0-analyzeProject.ts

```diff
@@ -152,6 +152,7 @@ export const analyzeProject = async (context: $TSContext): Promise<$TSContext> =
 const setProjectConfig = (context: $TSContext, projectName: string): void => {
   context.exeInfo.isNewProject = isNewProject(context);
   context.exeInfo.projectConfig = {
+    ...context.exeInfo.projectConfig,
```

Comment:
why was this added? was there a reason we didn't want to copy all properties from the config before?

Reply:
This was added because we *do* want to carry over all properties from the config before.